hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/ConversionUtils.java

```diff
@@ -66,6 +67,20 @@ public static List<ContractLoginfo> pbjLogsFrom(@NonNull final List<Log> logs) {
         return pbjLogs;
     }
 
+    /**
+     * Wraps the first 32 bytes of the given SHA-384 {@link com.swirlds.common.crypto.Hash hash} in a Besu {@link Hash}.
+     *
+     * @param sha384Hash the SHA-384 hash
+     * @return the first 32 bytes as a Besu {@link Hash}
+     */
+    public static org.hyperledger.besu.datatypes.Hash ethHashFrom(
+            @NonNull final com.hedera.pbj.runtime.io.buffer.Bytes sha384Hash) {
+        requireNonNull(sha384Hash);
+        final byte[] prefixBytes = new byte[32];
+        sha384Hash.getBytes(0, prefixBytes, 0, prefixBytes.length);
+        return org.hyperledger.besu.datatypes.Hash.wrap(Bytes32.wrap(prefixBytes));
```

Comment:
I don't _think_ there's any byte copying going on in here - just wrapping in other classes.  The copy takes place on the line above.  (And I'm not sure why even _that_ copy is needed.)